src/index.js

```diff
@@ -0,0 +1,76 @@
+const navSlide = () => {
+	const burger = document.querySelector('.burger');
+	const nav = document.querySelector('.nav-links');
+	const navLinks = document.querySelectorAll('.nav-links li');
+//adds class to burger that creates the animation
+	burger.addEventListener('click', () => {
+		nav.classList.toggle('nav-active');
+		navLinks.forEach((link, ind) => {
+			if (link.style.animation) {
+				link.style.animation = '';
+			} else {
+				link.style.animation = `navLinkFade 0.5s ease forwards ${ind / 7 + 1.0}s`;
+			}
+		});
+		burger.classList.toggle('toggle');
+	});
+};
+
+const editor = () => {
+	var TxtRotate = function(el, toRotate, period) {
+		this.toRotate = toRotate;
+		this.el = el;
+		this.loopNum = 0;
+		this.period = parseInt(period, 10) || 2000;
+		this.txt = '';
+		this.tick();
+		this.isDeleting = false;
+	};
+	
+	TxtRotate.prototype.tick = function() {
+		var i = this.loopNum % this.toRotate.length;
+		var fullTxt = this.toRotate[i];
+//i was gonna implement to delete but if it doesnt delete it adds the next letter
+		if (this.isDeleting) {
+			this.txt = fullTxt.substring(0, this.txt.length - 1);
+		} else {
+			this.txt = fullTxt.substring(0, this.txt.length + 1);
```

Comment:
My train of thought at first was that it would delete letter by letter by letter after the input. Than i thought that would take a year longer so i deleted it but yea that line should be changed since it has no functionality anymore